test/models/profile_test.rb

```diff
@@ -68,7 +68,16 @@ class ProfileTest < ActiveSupport::TestCase
 
     p = profiles(:one).dup
     assert_not p.update(policy_id: policies(:two).id, benchmark: bm)
-    assert  p.errors.full_messages.join['Policy type must be unique']
+    assert p.errors.full_messages.join['Policy type must be unique']
```

Comment:
I'm not sure why rubocop never complained about this before, but there was an extra space here.